android/src/main/java/io/radar/capacitor/RadarPlugin.java

```diff
@@ -455,6 +455,54 @@ public void onComplete(@NonNull Radar.RadarStatus radarStatus,
         });
     }
 
+    @PluginMethod
+    public void updateTrip(PluginCall call) {
+        JSObject optionsObj = call.getObject("options");
+        JSONObject optionsJson = RadarPlugin.jsonObjectForJSObject(optionsObj);
+        if (optionsJson == null) {
+            call.reject("options is required");
+            return;
+        }
+        RadarTripOptions options = RadarTripOptions.fromJson(optionsJson);
+        RadarTrip.RadarTripStatus status = RadarTrip.RadarTripStatus.UNKNOWN;
+        if (call.hasOption("status")) {
+            String statusStr = call.getString("status");
+            if (statusStr != null) {
+                if (statusStr.equalsIgnoreCase("started")) {
```

Comment:
These checks should be simplifiable to `RadarTrip.RadarTripStatus.forName(statusStr.toUpperCase(Locale.ROOT))`